neuror/main.py

```diff
@@ -22,8 +22,7 @@
 from nptyping import NDArray
 from scipy.spatial.distance import cdist
 
-from neuror import axon
-from neuror.cut_plane import CutPlane
```

Comment:
So, if at some point someone brings back this import, the problem reappears. From what I understand it is rather a bug of sphinx + autosummary.

Reply:
The doc will fail to build then, assuming that bug still exists